pkg/kubelet/cm/cpumanager/state/state_checkpoint.go

```diff
@@ -52,8 +52,7 @@ func NewCheckpointState(stateDir, checkpointName, policyName string) (State, err
 
 	if err := stateCheckpoint.restoreState(); err != nil {
 		//lint:ignore ST1005 user-facing error message
-		return nil, fmt.Errorf("could not restore state from checkpoint: %v\n"+
-			"Please drain this node and delete the CPU manager checkpoint file %q before restarting Kubelet.",
+		return nil, fmt.Errorf("could not restore state from checkpoint: %v, please drain this node and delete the CPU manager checkpoint file %q before restarting Kubelet",
```

Comment:
Why remove the newlines here? It is usually easer to read if it is broken up, and forces it to jump out when this is the error.

I'm not 100% against the change, just curious about the reasoning to change it.